ons-spark/raw-notebooks/checkpoint-staging/r_input.R

```diff
@@ -0,0 +1,159 @@
+options(warn = -1) 
+library(sparklyr)
+library(dplyr)
+library(DBI)
+
+sc <- sparklyr::spark_connect(
+    master = "local[2]",
+    app_name = "checkpoint",
+    config = sparklyr::spark_config())
+
+
+set.seed(42)
+new_cols <- 12
+num_rows <- 10^3
+
+start_time <- Sys.time()
+
+df = sparklyr::sdf_seq(sc, 1, num_rows) %>%
```

Comment:
All looks good - only last minor thing to change is just changing this to a <- assignment operator rather than an =, and making the same change on line 60 and 106 (sorry for not noticing these on my first review!)

Once you've made those changes I think it should be good to go - in terms of functionality it's all running okay, unless you've noticed anything off?